src/auto_archiver/modules/gsheet_feeder_db/__manifest__.py

```diff
@@ -51,6 +51,16 @@
             "help": "if True the stored files path will include 'workbook_name/worksheet_name/...'",
             "type": "bool",
         },
+        "must_have_folder_name_for_archive_to_run": {
+            "default": False,
+            "help": "if True the folder name must be set in the feeder eg DM001. This is to enforce files be stored in the correct folder on destination storage",
+            "type": "bool",
+        },
+        "allow_overwrite_of_spreadsheet_cells": {
+            "default": False,
+            "help": "if True spreadsheet cells in row will be overwritten if any data there already",
+            "type": "bool",
+        },
```

Comment:
this code is in another PR, ignoring here: https://github.com/bellingcat/auto-archiver/pull/304

Reply:
oops sorry about that.